### DISCUSSION TITLE
Year 2015 Not Working to Pull Data

### DISCUSSION THREAD

**Discussion by badger13** [cwendt94/espn-api#130] [General] [answered]
For some reason, my script is working to pull information for years 2014, 2016-2020, but the module will not work to pull 2015 data.  Here is a picture of the errors that show when I try to do 2015. I'm not super familiar with what all the error information means, so any assistance would be greatly appreciated! Thank you!
![Year 2015 Error Log](https://user-images.githubusercontent.com/34585276/94203514-28647880-fe85-11ea-9f41-e20ca177798b.png)

**Comment by cwendt94** [comment, marked as answer]
Someone reported a similar error yesterday and I was able to fix it in package version 0.7.1. Could you try updating to the latest package `0.7.1` and running it again? 

Thanks!

**Reply by badger13** [reply]
That worked. Thank you.
